### PR TITLE
Jonmv/data generator

### PR DESCRIPTION
**Pull request by jonmv**
@geirst please review.

### CODE REVIEW

**Comment by jonmv** [comment]
The word tally of the GPT-2 training data set is ~5Mb, which I believe is fine to check in. We could probably use a smaller data set as well.

**Comment by jonmv** [comment]
PTAL at the fast_access_attributes test for a taste of the new API :)

File: lib/generator.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'erb'
+
+class Generator
```
Comment (geirst) [inline lib/generator.rb:5]: Should we rename this to DataGenerator as well to match the java program?

File: lib/test/generator_test.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline lib/test/generator_test.rb:1]: Please update copyright.

File: tests/performance/fast_access_attributes/fast_access_attributes.rb [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Private reason: Depends on pub/ data
```
Comment (geirst) [inline tests/performance/fast_access_attributes/fast_access_attributes.rb:1]: Please replace this with copyright.

**Review by geirst — DISMISSED** [review]
👍

File: lib/generator.rb (hunk 1 above)
Comment (jonmv) [reply lib/generator.rb:5]: It's not just a DocumentGenerator, but I guess Data is fine ... :)

File: lib/test/generator_test.rb (hunk 2 above)
Comment (jonmv) [reply lib/test/generator_test.rb:1]:
```suggestion
# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

File: tests/performance/fast_access_attributes/fast_access_attributes.rb (hunk 3 above)
Comment (jonmv) [reply tests/performance/fast_access_attributes/fast_access_attributes.rb:1]:
```suggestion
# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**